Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -878,19 +943,19 @@ def _exec_editor_multitest(self, request: _pytest.fixtures.FixtureRequest,
         results = {}
         test_filenames_str = ";".join(editor_utils.get_testcase_module_filepath(test_spec.test_module) for test_spec in test_spec_list)
         cmdline = [
-            "--runpythontest", test_filenames_str,
+            test_run_python_command, test_filenames_str,
             "-logfile", f"@log@/{log_name}",
             "-project-log-path", editor_utils.retrieve_log_path(run_id, workspace)] + test_cmdline_args
 
-        editor.args.extend(cmdline)
-        editor.start(backupFiles = False, launch_ap = False, configure_settings=False)
+        test_launcher.args.extend(cmdline)
+        test_launcher.start(backupFiles=False, launch_ap=False, configure_settings=False)
 
         output = ""
         editor_log_content = ""
         try:
-            editor.wait(self.timeout_editor_shared_test)
-            output = editor.get_output()
-            return_code = editor.get_returncode()
+            test_launcher.wait(self.timeout_editor_shared_test)
+            output = test_launcher.get_output()
+            return_code = test_launcher.get_returncode()
             editor_log_content = editor_utils.retrieve_editor_log_content(run_id, log_name, workspace)
```

Comment:
all of these should be replaced with a generic function to retrieve the current test app log